ruby/vampires.rb

```diff
@@ -4,45 +4,70 @@
 sign_up = true
 result = "Result inconclusive."
 
-puts "What is your name?"
-name = gets.chomp
-name = name.downcase
-if (name == "drake cula") || (name == "tu fang")
-		real_name = false
-end
+puts "How many employees will be processed?"
+employees = gets.chomp
+x = 1
+until x > employees.to_i
 
-puts "How old are you? What year were you born?"
-age = gets.chomp
-year = gets.chomp
-if (2016 - year.to_i) != age.to_i
-	    real_age = false
-end
+	puts "What is your name?"
+	name = gets.chomp
+	name = name.downcase
+	if ( ( name == "drake cula" ) || ( name == "tu fang" ) )
+			real_name = false
+	end
 
-puts "Our company cafeteria serves garlic bread. Should we order some for you?(y/n)"
-order_bread = gets.chomp
-if (order_bread == "n")
-        garlic_bread = false
-end
+	puts "How old are you? What year were you born?"
+	age = gets.chomp
+	year = gets.chomp
+	if ( ( 2016 - year.to_i ) != age.to_i )

```

Comment:
with this setup you will have to change this file every year for it to work.

Check out the Ruby `Time` class. You should be able to handle a dynamic year check with something like `Time.new.year`